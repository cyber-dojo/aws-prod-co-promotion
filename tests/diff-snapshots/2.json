{
  "snappish1": {
    "snapshot_id": "aws-beta#4754",
    "artifacts": [
      {
        "fingerprint": "0fd1eae4a2ab75d4d08106f86af3945a9e95b60693a4b9e4e44b59cc5887fdd1",
        "name": "244531986313.dkr.ecr.eu-central-1.amazonaws.com/nginx:fa32058@sha256:0fd1eae4a2ab75d4d08106f86af3945a9e95b60693a4b9e4e44b59cc5887fdd1",
        "most_recent_timestamp": 1740904271,
        "flow": "nginx-ci",
        "commit_url": "https://github.com/cyber-dojo/nginx/commit/fa32058a046015786d1589e16af7da0973f2e726",
        "instance_count": 1
      },
      {
        "fingerprint": "337fa91d02fa59729aca2941bbfebf999d1c5ae74b1492a4c99a33a925c7f052",
        "name": "244531986313.dkr.ecr.eu-central-1.amazonaws.com/web:ed1878b@sha256:337fa91d02fa59729aca2941bbfebf999d1c5ae74b1492a4c99a33a925c7f052",
        "most_recent_timestamp": 1740904311,
        "flow": "web-ci",
        "commit_url": "https://github.com/cyber-dojo/web/commit/ed1878bd4aba3cada1e6ae7bc510f6354c61c484",
        "instance_count": 3
      }
    ]
  },
  "snappish2": {
    "snapshot_id": "aws-prod#3548",
    "artifacts": [
      {
        "fingerprint": "49cfb0d0696a9934e408ff20eaeea17ba87924ea520963be2021134814a086cc",
        "name": "244531986313.dkr.ecr.eu-central-1.amazonaws.com/web:7645ee7@sha256:49cfb0d0696a9934e408ff20eaeea17ba87924ea520963be2021134814a086cc",
        "most_recent_timestamp": 1740900522,
        "flow": "web-ci",
        "commit_url": "https://github.com/cyber-dojo/web/commit/7645ee70706ad259c6367b97946edb10d00815a0",
        "instance_count": 3
      },
      {
        "fingerprint": "0f803b05be83006c77e8c371b1f999eaabfb2feca9abef64332633362b36ca94",
        "name": "244531986313.dkr.ecr.eu-central-1.amazonaws.com/nginx:e92d83d@sha256:0f803b05be83006c77e8c371b1f999eaabfb2feca9abef64332633362b36ca94",
        "most_recent_timestamp": 1740406956,
        "flow": "nginx-ci",
        "commit_url": "https://github.com/cyber-dojo/nginx/commit/e92d83d1bf0b1de46205d5e19131f1cee2b6b3da",
        "instance_count": 1
      }
    ]
  },
  "changed": {
    "artifacts": []
  },
  "not-changed": {
    "artifacts": [
      {
        "fingerprint": "2c6079df58292ed10e8074adcb74be549b7f841a1bd8266f06bb5c518643193e",
        "name": "244531986313.dkr.ecr.eu-central-1.amazonaws.com/exercises-start-points:86f9052@sha256:2c6079df58292ed10e8074adcb74be549b7f841a1bd8266f06bb5c518643193e",
        "most_recent_timestamp": 1740902980,
        "flow": "exercises-start-points-ci",
        "commit_url": "https://github.com/cyber-dojo/exercises-start-points/commit/86f9052b15e5b5a7f4b5811377f5d2a7158d11f3",
        "instance_count": 1
      },
      {
        "fingerprint": "80798fa02d01e0b219d137d5b3fa7c83bcd6299eb1c8f7c4f7a5f1c83cc346bc",
        "name": "244531986313.dkr.ecr.eu-central-1.amazonaws.com/saver:1faf96b@sha256:80798fa02d01e0b219d137d5b3fa7c83bcd6299eb1c8f7c4f7a5f1c83cc346bc",
        "most_recent_timestamp": 1740900233,
        "flow": "saver-ci",
        "commit_url": "https://github.com/cyber-dojo/saver/commit/1faf96b7b153d2a24f092e36831741cfbaff70ea",
        "instance_count": 1
      },
      {
        "fingerprint": "9ac8b25299e40c1f1a6a4c29d52fcbbfdcb7d9bbcd7301747c70fcb3f3d68201",
        "name": "244531986313.dkr.ecr.eu-central-1.amazonaws.com/dashboard:72fe389@sha256:9ac8b25299e40c1f1a6a4c29d52fcbbfdcb7d9bbcd7301747c70fcb3f3d68201",
        "most_recent_timestamp": 1740898219,
        "flow": "dashboard-ci",
        "commit_url": "https://github.com/cyber-dojo/dashboard/commit/72fe389434d00ae28f12a33810abbb812affc730",
        "instance_count": 1
      },
      {
        "fingerprint": "9cd1b41f3379e4a48b6421c7ddfa531d291addb88909eb1cd352f3212aa948af",
        "name": "244531986313.dkr.ecr.eu-central-1.amazonaws.com/runner:6e98a42@sha256:9cd1b41f3379e4a48b6421c7ddfa531d291addb88909eb1cd352f3212aa948af",
        "most_recent_timestamp": 1740902953,
        "flow": "runner-ci",
        "commit_url": "https://github.com/cyber-dojo/runner/commit/6e98a42a765dcd88aeb95402fa25ced1608b572c",
        "instance_count": 3
      },
      {
        "fingerprint": "dd123b623e731fdccfc5343ca30f927ae5f58df443fa15f00c496427a6e14eea",
        "name": "244531986313.dkr.ecr.eu-central-1.amazonaws.com/creator:dca5d2f@sha256:dd123b623e731fdccfc5343ca30f927ae5f58df443fa15f00c496427a6e14eea",
        "most_recent_timestamp": 1740394746,
        "flow": "creator-ci",
        "commit_url": "https://gitlab.com/cyber-dojo/creator/-/commit/dca5d2f7571f9b63d651088c2b38946091853083",
        "instance_count": 1
      },
      {
        "fingerprint": "e3c25b3a0f32f1a93662d81e14312f82cd08ab7c16b230fcd1fb1d92106efd9e",
        "name": "244531986313.dkr.ecr.eu-central-1.amazonaws.com/differ:ed56bc1@sha256:e3c25b3a0f32f1a93662d81e14312f82cd08ab7c16b230fcd1fb1d92106efd9e",
        "most_recent_timestamp": 1740407304,
        "flow": "differ-ci",
        "commit_url": "https://github.com/cyber-dojo/differ/commit/ed56bc1e9deb5df3de2cb1c4cfd47cb3b58c7136",
        "instance_count": 1
      }
    ]
  }
}
