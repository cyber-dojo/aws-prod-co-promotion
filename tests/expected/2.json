[
  {
    "incoming_image_name": "244531986313.dkr.ecr.eu-central-1.amazonaws.com/nginx:fa32058@sha256:0fd1eae4a2ab75d4d08106f86af3945a9e95b60693a4b9e4e44b59cc5887fdd1",
    "incoming_fingerprint": "0fd1eae4a2ab75d4d08106f86af3945a9e95b60693a4b9e4e44b59cc5887fdd1",
    "incoming_repo_url": "https://github.com/cyber-dojo/nginx",
    "incoming_repo_name": "nginx",
    "incoming_commit_sha": "fa32058a046015786d1589e16af7da0973f2e726",
    "incoming_flow": "nginx-ci",
    "outgoing_image_name": "244531986313.dkr.ecr.eu-central-1.amazonaws.com/nginx:e92d83d@sha256:0f803b05be83006c77e8c371b1f999eaabfb2feca9abef64332633362b36ca94",
    "outgoing_fingerprint": "0f803b05be83006c77e8c371b1f999eaabfb2feca9abef64332633362b36ca94",
    "outgoing_repo_url": "https://github.com/cyber-dojo/nginx",
    "outgoing_repo_name": "nginx",
    "outgoing_commit_sha": "e92d83d1bf0b1de46205d5e19131f1cee2b6b3da",
    "outgoing_flow": "nginx-ci",
    "deployment_diff_url": "https://github.com/cyber-dojo/nginx/compare/e92d83d1bf0b1de46205d5e19131f1cee2b6b3da...fa32058a046015786d1589e16af7da0973f2e726"
  },
  {
    "incoming_image_name": "244531986313.dkr.ecr.eu-central-1.amazonaws.com/web:ed1878b@sha256:337fa91d02fa59729aca2941bbfebf999d1c5ae74b1492a4c99a33a925c7f052",
    "incoming_fingerprint": "337fa91d02fa59729aca2941bbfebf999d1c5ae74b1492a4c99a33a925c7f052",
    "incoming_repo_url": "https://github.com/cyber-dojo/web",
    "incoming_repo_name": "web",
    "incoming_commit_sha": "ed1878bd4aba3cada1e6ae7bc510f6354c61c484",
    "incoming_flow": "web-ci",
    "outgoing_image_name": "244531986313.dkr.ecr.eu-central-1.amazonaws.com/web:7645ee7@sha256:49cfb0d0696a9934e408ff20eaeea17ba87924ea520963be2021134814a086cc",
    "outgoing_fingerprint": "49cfb0d0696a9934e408ff20eaeea17ba87924ea520963be2021134814a086cc",
    "outgoing_repo_url": "https://github.com/cyber-dojo/web",
    "outgoing_repo_name": "web",
    "outgoing_commit_sha": "7645ee70706ad259c6367b97946edb10d00815a0",
    "outgoing_flow": "web-ci",
    "deployment_diff_url": "https://github.com/cyber-dojo/web/compare/7645ee70706ad259c6367b97946edb10d00815a0...ed1878bd4aba3cada1e6ae7bc510f6354c61c484"
  }
]
